scripts/tasks/postprocess-commonjs.js

```diff
@@ -0,0 +1,29 @@
+module.exports.postprocessCommonjsTask = function() {
+  const { mod } = require('riceburn');
+  const ts = require('typescript');
+  const getAllPackageInfo = require('../monorepo/getAllPackageInfo');
+  const allPackages = Object.keys(getAllPackageInfo());
+
+  mod('lib-commonjs/**/*.js').asTypescript((node, modder) => {
+    if (ts.isCallExpression(node)) {
+      if (node.expression.getText() === 'require' && node.arguments.length === 1) {
+        const arg = node.arguments[0];
+        if (ts.isStringLiteral(arg)) {
+          if (belongsToFabric(arg.text) && arg.text.includes('lib/')) {
+            modder.replace(arg, `"${arg.text.replace('lib', 'lib-commonjs')}"`);
+          }
+        }
+      }
+    }
+  });
+
+  function belongsToFabric(requireArg) {
```

Comment:
I don't understand what this is doing or how it relates to fabric. Prefer no "fabricisms" in this script if possible :)

Reply:
I think this is filtering out imports that are in the local repo

Reply:
Well, this should only convert lib-commonjs for fabric related packages so inherently fabric related. This function picks up all the local packages as something we would postprocess for a file.